pyapr/data_containers/src/BindParameters.hpp

```diff
@@ -17,7 +17,6 @@ void AddPyAPRParameters(pybind11::module &m) {
         .def_readwrite("rel_error", &APRParameters::rel_error)
         .def_readwrite("sigma_th", &APRParameters::sigma_th)
         .def_readwrite("sigma_th_max", &APRParameters::sigma_th_max)
-        .def_readwrite("extra_smooth", &APRParameters::extra_smooth)
```

Comment:
What was this parameter?

Reply:
It added an additional round of smoothing of the local intensity scale. Something experimental that made it into the library, but it is now removed from LibAPR

Reply:
Yeah; was this me? and migrated through when experimenting with APR + t and recursive APR's.

Reply:
I think we added this during my master's in an attempt to better handle natural images. Not really sure, but I am sure it can be safely removed :sweat_smile: it's already gone on `LibAPR/develop`